cmd/zdb/zdb.c

```diff
@@ -6566,6 +6566,57 @@ zdb_read_block(char *thing, spa_t *spa)
 	else
 		zdb_dump_block(thing, buf, size, flags);
 
+	if ((flags & ZDB_FLAG_CHECKSUM) && !(flags & ZDB_FLAG_RAW) &&
```

Comment:
Maybe a 'nit, but a short comment description what this block of code is going to do might be nice...

Reply:
Done!